bridge/launchpad/import.go

```diff
@@ -70,6 +73,26 @@ func (li *launchpadImporter) ImportAll(repo *cache.RepoCache) error {
 			fmt.Println("TODO: Update bug")
 		}
 
+		/* Handle messages */
+		// The Launchpad API returns the bug description as the first
+		// comment, so skip it.
+		for _, lpMessage := range lpBug.Messages[1:] {
+			createdAt, _ := time.Parse(time.RFC3339, lpMessage.CreatedAt)
+			err = b.AddCommentRaw(
+				li.makePerson(lpMessage.Owner),
+				createdAt.Unix(),
+				lpMessage.Content,
+				nil,
+				nil)
+			if err != nil {
+				return errors.Wrapf(err, "failed to add comment to bug #%s", lpBugID)
+			}
+			err = b.CommitAsNeeded()
```

Comment:
So, I'm not sure why I need to "manually" commit the comments, when the bug itself is automagically committed. I would to get an explanation :)

Reply:
It's the other way around. All new edit on a bug pile up in a staging area, kind of like git (see https://github.com/MichaelMure/git-bug/blob/master/bug/bug.go#L64). These changes are visible by any UI but are not yet written in git. Once it's time to write all that (this time is determined by each UI), a `Commit()` will write all edits in a single commit. `CommitAsNeeded()` is the same thing but only commit if there is actually something to commit so you don't get an error. All that is meant to limit the number of commit and improve the perfomance, be it reading speed or network transfer.

Now, the tricky thing is that the cache need to know the bug id to store a new bug. But a bug only get it's id when it's commited the first time (it's the hash of the first commit). That's why creating a new bug through the cache automatically commit it.

Reply:
So I should commit as little as possible, right? In this case, I should probably only commit when returning from the function, am I right?

Reply:
That's correct. The github importer commit at the end of each individual bug import.

Reply:
Yes there is a way. Bugs, comment and everything you import should be tagged in the metadata with the corresponding launchpad id. That way, you can look in the bug edition for any operation with the id you want to import so see if it's already there or not.

in the github importer:
lookup --> https://github.com/MichaelMure/git-bug/blob/master/bridge/github/import.go#L390
tagging --> https://github.com/MichaelMure/git-bug/blob/master/bridge/github/import.go#L448

Reply:
OK. Launchpad comments do not have their own ID, per say. I'll probably end up using "<bug-id>-<comment-index>" if this identifier is just used internally.

Reply:
You can use anything as long as its unique and stable. Maybe an url ? a made up id from the bug id and a comment index ? a hash of the content ?

Reply:
It's not used internally. Metadata are for your own usage, that's why they are prefixed with `launchpad-`